systemtest/approvals/TestRUMErrorSourcemapping/absolute_bundle_filepath/standalone.approved.json

```diff
@@ -27,192 +30,6 @@
         "error.exception.message": [
             "Uncaught Error: timeout test error"
         ],
-        "error.exception.stacktrace": [
```

Comment:
The mapping changes will need to be documented somewhere; I consider them bugfixes as they result from problems that we have observed with using `flattened` here, but users might have built on top of it already.

Reply:
:+1: I will add a note to the changelog